OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3566,13 +3567,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <desc>Fine and regular handwriting. Regularly spaced characters.</desc>
                            <seg type="ink">Black, red.</seg>
                                 <seg type="rubrication">
-                                    Incipits (one word to one or several lines, consecutive or alternating with black lines in different groupings), 
-                                    elements of punctuation signs, numerals and text dividers,
-                                    holy names (<persName ref="PRS6819Mary"/>, <persName ref="PRS4377Gabriel"/>, 
-                                    rarely <persName ref="PRS5684JesusCh"/>), the names of individuals related to the production of 
+                                    Incipits (one word to one or several lines, consecutive or alternating with black lines in different groupings); 
+                                    captions of miniatures; holy names (<persName ref="PRS6819Mary"/>, <persName ref="PRS4377Gabriel"/>, 
+                                    rarely <persName ref="PRS5684JesusCh"/>); the names of individuals related to the production of 
                                     the manuscript (<persName ref="PRS12911KidanaK"/>,
                                     <persName ref="PRS13031WaldaA"/>, <persName ref="PRS12964Gorgoryos"/>
-                                    <persName ref="PRS13026KenfaM"/>), and captions of miniatures.
+                                    <persName ref="PRS13026KenfaM"/>); numerals; text dividers; elements of punctuation signs.
                                 </seg>
                                 <list type="abbreviations">
                                     <item><abbr>አ</abbr>, <abbr>አሜ</abbr> for <expan>አሜን</expan> (e.g., on <locus target="#9rb #13ra #93ra #96vb"/>);</item>
```

Comment:
no comma after e.g. (@SusanneHummel ?)

Reply:
poor susanne :( I've added this to the guidelines document in the hope that she does not have to confirm comma use with every new PR...